src/GUI/RevEng.Shared/TableInformationModel.cs

```diff
@@ -19,6 +20,7 @@ public class TableInformationModel : INotifyPropertyChanged
         private bool _hasPrimaryKey;
         private bool _showKeylessWarning;
         private ObjectType _objectType;
+        private IEnumerable<string> _excludedColumns = new List<string>();
```

Comment:
Will this add empty ExcludedColumns arrays to all tables when serialized?

Reply:
yes

Reply:
Could we avoid that?

Reply:
I removed initialization and added EmitDefaultValues = false to DataMember. It still serializes the null value.
```
{
  ExcludedColumns: null
   ...
}
```

It shouldn't be a problem, but I'll keep investigating.

Reply:
Never mind...